WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderTransactionAlertEmailReceiptAction.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import MessageUI
+
+enum CardReaderTransactionAlertEmailReceiptAction {
+    case emailSent(String)
+    case sendEmail(() -> Void)
```

Comment:
What do you think if we add descriptors to the associated values? I wasn't sure at first glance what the String type was until I checked the context where it was called. Sort of:

```suggestion
    case emailSent(_ emailAddress: String)
    case sendEmail(_ onSendAction: () -> Void)
```

Reply:
Sounds good!